code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -673,6 +673,13 @@
 	walk_to(src, target, minimum_distance, delay)
 
 /mob/living/simple_animal/hostile/proc/AttackingTarget(atom/attacked_target)
+	if(client)
```

Comment:
This is necessary for re-saning. FF is a feature, not a bug

Reply:
This is only for abnormalities, I don't think abnormalities should be able to re-sane in R-Corp.